bootstrap/kubeadm/internal/bottlerocket/bottlerocket.go

```diff
@@ -156,6 +163,18 @@ func getBottlerocketNodeUserData(bootstrapContainerUserData []byte, users []boot
 	return bottlerocketNodeUserData, nil
 }
 
+func parseTaints(taints []corev1.Taint) string {
+	if len(taints) == 0 {
+		return ""
+	}
+	var taintsToml strings.Builder
+	for _, taint := range taints {
+		taintsToml.WriteString(fmt.Sprintf("\"%v\" = [\"%v:%v\"]", taint.Key, taint.Value, taint.Effect))
```

Comment:
This should be an array of more than one value right? Based on the documentation on Bottlerocket I see the following as a possibility
```
"dedicated" = ["experimental:PreferNoSchedule", "experimental:NoExecute"]
```

https://github.com/bottlerocket-os/bottlerocket#kubernetes-settings

Reply:
done, retested with multiple taints with the same key value; works